tests/lib/cli-engine.js

```diff
@@ -2301,14 +2305,17 @@ describe("CLIEngine", () => {
                 });
 
                 // create a new spy
-                spy = sandbox.spy(fs, "readFileSync");
+                // eslint-disable-next-line no-underscore-dangle
+                spy = sandbox.spy(engine._lintResultCache, "reconcile");
```

Comment:
Here, cause changes in `file-entry-cache` that all use `readFileSync` right now, but do you have any good ideas about it?

Reply:
I prefer to revert back to the original behavior that didn't depend on this functionality. I can make the change in `file-entry-cache` so the checksum check is optional, so the original tests won't need to change much.